draft/draft-ietf-tcpm-yang-tcp.xml

```diff
@@ -82,25 +82,26 @@
   <middle>
     <section title="Introduction">
       <t>The <xref target="I-D.ietf-tcpm-rfc793bis">Transmission Control
-      Protocol (TCP)</xref> is used by many applications in the Internet, including control
-      and management protocols. As such, TCP is implemented on network
-      elements that can be configured via network management protocols such as
+      Protocol (TCP)</xref> is used by many applications in the Internet,
+      including controland management protocols. As such, TCP is
+      implemented on network elements that can be configured and managed
+      via network management protocols such as
       <xref target="RFC6241">NETCONF</xref> or <xref
       target="RFC8040">RESTCONF</xref>.</t>
 
       <t>This document specifies a minimal <xref target="RFC7950">YANG
       1.1</xref> model for configuring and managing TCP on network
       elements that support YANG, a TCP connection table,
-      a TCP listner table containing
-      information about a particular TCP listner, and an augmentation
+      a TCP listener table containing
+      information about a particular TCP listener, and an augmentation
       of the <xref target="RFC8177">YANG Data Model for Key
-      Chains</xref> to support authentication. This YANG module is
+      Chains</xref> to support authentication. That YANG module is
```

Comment:
What is "That" referring to? With "This", it is clear we are talking about the YANG model in the draft.

Reply:
Done